pkg/util/util.go

```diff
@@ -132,6 +132,21 @@ func ContainsString(stringarr []string, element string) bool {
 	return false
 }
 
+// ListAMinusListB returns list of string which are in listA but not in listB
+func ListAMinusListB(listA []string, listB []string) []string {
```

Comment:
I would like to grab some attention on the function name here. ( Optional )
One suggestion : 
`ListDiff`

Reply:
I will update it

Reply:
Done